calender.rb

```diff
@@ -0,0 +1,75 @@
+# global variables
+$days_abbrevations = %w[S M T W T F S]
+$days_start_index = {"Sunday" => 0, "Monday" => 1, "Tuesday" => 2, "Wednesday" => 3, "Thursday" => 4, "Friday" => 5, "Saturday" => 6}
+#dateObj.strftime('%A')
+
+class Calender
+    def initialize()
```

Comment:
no need of parenthesis if no args are being passed.
 ```suggestion
    def initialize
```